src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -316,34 +347,25 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
     link: function (scope, element) {
       var setupDonutChartTitle = function () {
         $timeout(function () {
-          var donutChartTitle, bigText, smText;
+          var donutChartTitle, centerLabelText;
 
           donutChartTitle = element[0].querySelector('text.c3-chart-arcs-title');
-          if (scope.config.centerLabelFn) {
-            donutChartTitle.innerHTML = scope.config.centerLabelFn(scope);
-          } else if (scope.centerLabel === 'none') {
-            donutChartTitle.innerHTML = '';
+          if (!donutChartTitle) {
+            return;
+          }
+
+          centerLabelText = scope.getCenterLabelText(scope);
+
+          if (scope.config.centerLabelFn || scope.centerLabel === 'none') {
+            donutChartTitle.innerHTML = centerLabelText.bigText;
```

Comment:
I want 'getCenterLabelText' to return all possible label values (for unit testing).  If using a function or 'none', then we set the entire innerHTML to whatever centerLabelText.bigText has been set to; any other label format will divide the data into bigText and smText, to be used in the HTML spans specified.